docs/tutorials/using_dpdk.md

```diff
@@ -0,0 +1,632 @@
+
+| Machine                              | CPUs | RAM | Disk | Networks                       |
+|--------------------------------------|------|-----|------|--------------------------------|
+| User Plane Kubernetes Cluster        | 4    | 12g | 20g  | `management`, `access`, `core` |
+
+Create VM with Multipass:
+
+```console
+multipass launch -c 4 -m 12G -d 20G -n user-plane  --network mgmt-br --network core-br --network access-br jammy
+```
+
+Wait until all the VM is in a `Running` state.
+
+### Checkpoint 1: Is the VM ready ?
+
+You should be able to see the VM in a `Running` state with its default IP addresses by executing the following command:
+
+```shell
+multipass list
+```
+
+The output should be similar to the following:
+
+```
+Name                    State             IPv4             Image
+user-plane              Running           10.64.239.201    Ubuntu 22.04 LTS
+```
+
+## 3. Prepare the Virtual Machine
+
+Log in to the `user-plane` VM:
+
+```shell
+multipass shell user-plane
+```
+
+### Enable HugePages in the User Plane VM
+
+Update Grub to enable 2 units of 1Gi HugePages in the User Plane VM. Then, the VM is gracefully rebooted to activate the settings.
+
+```shell
+sudo sed -i "s/GRUB_CMDLINE_LINUX=.*/GRUB_CMDLINE_LINUX='default_hugepagesz=1G hugepages=2'/" /etc/default/grub
+sudo update-grub
+sudo init 6
+```
+
+#### Checkpoint 2: Are HugePages enabled ?
+
+You should be able to see the 2 units of Free HugePages with 1048576 kB size by executing the following command:
+
+```shell
+ cat /proc/meminfo | grep Huge
+```
+
+The output should be similar to the following:
+
+```shell
+AnonHugePages:         0 kB
+ShmemHugePages:        0 kB
+FileHugePages:         0 kB
+HugePages_Total:       2
+HugePages_Free:        2
+HugePages_Rsvd:        0
+HugePages_Surp:        0
+Hugepagesize:    1048576 kB
+Hugetlb:         2097152 kB
+```
+
+### Configure IP addresses in the User Plane VM
+
+Configure IP addresses for `mgmt`, `core` and `access` interfaces and add route from `access` to `ran` network by replacing the content of `/etc/netplan/50-cloud-init.yaml`:
+
+```shell
+cat << EOF | sudo tee /etc/netplan/50-cloud-init.yaml
+network:
+    ethernets:
+        enp5s0:
+            dhcp4: true
+        enp6s0:
+            dhcp4: false
+            addresses:
+              - 10.201.0.102/24
+            optional: true
+        enp7s0:
+            dhcp4: false
+            addresses:
+              - 10.203.0.100/24
+            optional: true
+        enp8s0:
+            dhcp4: false
+            addresses:
+              - 10.202.0.100/24
+            routes:
+              - to: 10.204.0.0/24
+                via: 10.202.0.1
+            optional: true
+    version: 2
+EOF
+```
+
+Apply the network configuration:
+
+```shell
+sudo chmod 600 /etc/netplan/50-cloud-init.yaml
+sudo netplan apply
+```
+
+### Create the MACVLAN bridges for `enp7s0` and `enp8s0`
```

Comment:
This is not needed. When we unbind using `driverctl`, these interfaces go away. The `diverctl` and `enable_unsafe_noiommu_mode` could perhaps be put instead into `/etc/rc.local`

Reply:
Create the MACVLAN bridges part is removed and other configurations are persisted using `/etc/rc.local`.

Reply:
Done